docs/simulation_models.rst

```diff
@@ -2,4 +2,207 @@
+through the pipes). The simulation model evaluates a steady state of the hydraulic and thermal
+physical equations. This also means that consecutive time steps are modelled independently and the
+behaviour of thermal storages cannot be represented. A dynamic simulation model may be implemented
+at a later point in time.
+
+
+Usage
+-----
+
+.. code-block:: txt
+
+    tree
+    ├── consumers.csv
+    ├── edges.csv
+    ├── forks.csv
+    ├── producers.csv
+    └── sequences
+        ├── consumers-mass_flow.csv
+        └── consumers-temperature_drop.csv
+
+.. code-block:: python
+
+    import dhnx
+
+    tnw = dhnx.network.ThermalNetwork()
+
+    tnw.simulate()
+
+
+.. 	figure:: _static/radial_network_details.svg
+   :width: 70 %
+   :alt: radial_network_details.svg
+   :align: left
+
+   Fig. 1: Schematic
+
+
+
+
+
+Underlying Concept
+------------------
+
+.. _simulation_model_table_label:
+
+.. csv-table::
+   :header-rows: 1
+   :delim: ;
+   :file: _static/simulation_models.csv
+
+
+
+The following equations are related to a single pipe.
+
+Hydraulic equations
+~~~~~~~~~~~~~~~~~~~
+
+A pressure difference between two ends of a pipe occurs because of three effects:
+
+* distributed pressure losses along the pipe's inner surface
+* local pressure losses at distinct items,
+* hydrostatic pressure differences because of a difference in height.
+
+All three effects can be captured in this formula:
+
+.. math::
+    \Delta p = \Delta p_{loc} + \Delta p_{dis} + \Delta p_{hydrostatic}
+
+**Distributed pressure losses**
+
+Darcy-Weissbach-equation discribes distributed pressure losses
+:math:`\Delta p_{dis}` inside the pipes as
+
+.. math::
+    \Delta p_{dis} = \lambda \rho \frac{L}{2D} v^2.
+
+
+Together with the flow velocity
+
+.. math::
+    v = \frac{4 \dot{m}}{\rho \pi D^2}
+
+this can be written to
+
+.. math::
+    \Delta p_{dis} = \lambda \frac{8 L}{\rho \pi^2 D^5} \dot{m}^2.
+
+
+Here, :math:`\lambda = \lambda(Re, \epsilon, D)`, depends on
+:math:`Re:`, der Rohrrauigkeit (pipe's surface roughness) :math:`\epsilon` und dem Rohrdurchmesser
+:math:`D`.
+The Reynolds number
+
+.. math:::
+    Re = \frac{Dv\rho}{\mu}
+
+
+is a dimensionless quantity characterizing fluid flows. :math:`\mu` is the dynamic viscosity of
+water.
+
+In a pipe, flow is laminar if :math:`Re` < 2300 and turbulent if :math:`Re` > 4000.
+In district heating pipes, flow is usually turbulent. The turbulent flow regime can be further
+distinguished into smooth, intermediate and rough regime depending on the pipe's surface roughness.
+
+:math:`\lambda` can be calculated using approximation formulas. Bordin2015 nimmt folgende Formel an:
+
+.. math::
+    \lambda = 0.07 \cdot Re ^{-0.13} \cdot D^{-0.14}.
+
+A more accurate approximation of the Colebrook-White-equation for flow in pipes is given by this
+formula:
+
+.. math::
+    \lambda = \frac{1.325}{(ln(\frac{\epsilon}{3.7D} + \frac{5.74}{Re^{0.9}}))^2}.
```

Comment:
Would be nice. Do we have one?

Reply:
So far I found the equation only on this website: https://swmm5.org/2017/11/07/moody-friction-factor-calculator-from-innovyze-h2ocalc/

I checked the reference (Swamee, D.K. and Jain, A.K. (1976) Explicit Equations for Pipe Flow Problems. Journal of the Hydraulics Division, 102, 657-664.) but there the equation differs (it has the decimal logarithm instead of natural logarithm).